### DISCUSSION TITLE
Using Liferay's out of the box features

### DISCUSSION THREAD

**Discussion by monicaAzizAbdo** [TDRA-ae/aegov-dls#77] [General]
I'm currently implementing page comments feature for a project using Liferay
The thing is we're using the out of the box feature to implement this feature which means that we have no access to the code structure...
The question is, is it acceptable to utilize your own styles to align the mentioned feature with the DLS look and feel (without using DLS' classes)

**Comment by moiz-conceptlz** [comment]
Hello @monicaAzizAbdo 

Yes, it is acceptable to style out-of-the-box features provided by plugins or embedded components to match the style of the design system. 

However, please be informed that if there is a change in the design system component and style, you would be responsible to make similar updates to your CSS.
